spark/src/test/scala/io/delta/sharing/spark/RemoteDeltaLogSuite.scala

```diff
@@ -66,6 +66,7 @@ class RemoteDeltaLogSuite extends SparkFunSuite with SharedSparkSession {
 
   test("RemoteSnapshot getFiles with limit and jsonPredicateHints") {
     val spark = SparkSession.active
+    spark.sessionState.conf.setConfString("spark.delta.sharing.jsonPredicateHints.enabled", "true")
```

Comment:
Shall we add a test when we set this to false.

Reply:
Saw this a bit late... good idea: sent another [ PR ] (https://github.com/delta-io/delta-sharing/pull/273)